backend-rust/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to this project will be documented in this file.
 
 Database schema version: 10
 
+- Validation on rows affected on bakers only occurs after protocol version 7 is in effect. 
```

Comment:
- Relaxing validation constraint on database rows affecting bakers prior to protocol version 7.